docs/mechanical/Gears.md

```diff
@@ -0,0 +1,219 @@
+# Gears
+
+
+## What are Gears?
+
+
+  
+*   Gears are defined as toothed element which are used for transmitting rotary motion from one shaft to another.
+*   Gears are most often used in transmissions to convert an electric motor’s high speed and low torque to a shaft’s requirements for low speed high torque.
+*   Gears essentially allow positive engagement between Teeth, so high forces can be transmitted while still undergoing essentially rolling contact.
+*   Gears do not depend on friction as belt drives and do best when friction is minimized.
+*   The motion and power transmitted by gears is kinematically equivalent to that transmitted by friction wheels or discs.
+
+  
+
+![gears1](images/gears1.png)
+
+_Transmission of Power_
+
+  
+
+*   In order to avoid the slipping, a number of projections (called teeth) as shown in Fig. (b), are provided on the periphery of the wheel A, which will fit into the corresponding recesses on the  periphery of the wheel B. A friction wheel with the teeth cut on it is known as **_toothed wheel or gear._** **_The usual connection to show the toothed wheels is by their pitch circles_**
+
+  
+
+![gears2](images/gears2.png)
+
+  
```

Comment:
Can you resize this properly so that the image is not pixalated? The <img> works now